### PR TITLE
[Console] [MyAccount] Add New Product-IS Logo and Remove Static Product Name from Header Component

### PR DESCRIPTION
**Pull request by yasinmiran**
## Purpose
Please note $subject

### CODE REVIEW

File: apps/console/src/public/deployment.config.json [hunk 1]
```diff
@@ -309,7 +309,7 @@
         },
         "isGroupAndRoleSeparationEnabled": true,
         "isClientSecretHashEnabled": false,
-        "productName": "Identity Server",
+        "productName": "WSO2 Identity Server",
```
Comment (brionmario) [inline apps/console/src/public/deployment.config.json:312]: Let's do the same change for myaccount config.

Comment (yasinmiran) [reply apps/console/src/public/deployment.config.json:312]: Missing configurations in `myaccount` application is added in commit 6f5d9f8. Thanks!

**Comment by yasinmiran** [comment]
@brionmario below is the SVG image. As a reference, this SVG has a white background and may not support the dark theme in future. We have to remove it and make it transparent.

Screenshot from 2021-01-15 14-58-35

**Comment by brionmario** [comment]
> @brionmario below is the SVG image. As a reference, this SVG has a white background and may not support the dark theme in future. We have to remove it and make it transparent.
> 
> Screenshot from 2021-01-15 14-58-35

ATM, we are maintaining theme wise assets, so fo `dark` theme, there will be an asset bundle.

Regarding the logo SVG, let's make it transparent.

**Review by brionmario — APPROVED** [review]

**Pull request merged**